config/initializers/config.rb

```diff
@@ -24,6 +24,8 @@
   'RAILS_CACHE_REDIS_PASSWORD',
   'SIDEKIQ_CONCURRENCY',
   'ANALYTICS_DASHBOARD_CACHE_TTL',
+  "USER_AUTH_LOCKOUT_IN_MINUTES",
+  "USER_AUTH_MAX_FAILED_ATTEMPTS"
```

Comment:
If you go with the ["optional configuration" approach](https://github.com/simpledotorg/simple-server/pull/975/files#r428483384), we can remove these two from here.